upload_package

```diff
@@ -54,13 +54,14 @@ PAYLOAD_SIZE=$(stat --format='%s' "${UPDATE_PATH}")
 PAYLOAD_SHA1=$(cat "${UPDATE_PATH}" | openssl dgst -sha1 -binary | base64)
 PAYLOAD_SHA256=$(cat "${UPDATE_PATH}" | openssl dgst -sha256 -binary | base64)
 
-sha256sum "${UPDATE_PATH}" > "${UPDATE_CHECKSUM_PATH}"
+(cd "${DATA_DIR}" && sha256sum "flatcar_production_update.gz") > "${UPDATE_CHECKSUM_PATH}"
```

Comment:
Not really a change request. Another way of doing it could be:

```suggestion
env --chdir "${DATA_DIR}" sha256sum "flatcar_production_update.gz" > "${UPDATE_CHECKSUM_PATH}"
```

Just something I found out some time ago.

Reply:
Looks sleek^^